lib/lita/handlers/totems.rb

```diff
@@ -1,10 +1,15 @@
 require 'lita'
 require 'chronic_duration'
 require 'redis-semaphore'
+require 'signalfx'
 
 module Lita
   module Handlers
     class Totems < Handler
+      
+      def signalfx_client
+        SignalFx.new ENV['LITA_SLACK_TOKEN']
```

Comment:
@mdimas @quaelin I want to have only one SignalFX instance to use for any call, wonder if this is the way to do it?
was trying to search online and seems to be the use of `@` but it doesnt seem to like it

Reply:
I think you could do something like this:
```ruby
  def signalfx_client
    @signalfx_client ||= SignalFx.new ENV['LITA_SLACK_TOKEN']
  end
```

The `||=` means it'll only invoke `SignalFx.new` as long as `@signalfx_client` doesn't already have a value.

Reply:
Thanks!!